client/python/kserve-api/samples/grpc_async_infer_resnet.py

```diff
@@ -0,0 +1,203 @@
+#

```

Comment:
we should preserve original header if this sample is based on the triton samples